tests/test_downloader_handlers.py

```diff
@@ -437,6 +437,8 @@ class S3AnonTestCase(unittest.TestCase):
         import boto
     except ImportError:
         skip = 'missing boto library'
+    if six.PY3:
+        skip = 'S3 not supported on Py3'
```

Comment:
boto declares Python 3 support in its setup.py - see https://github.com/boto/boto/blob/develop/setup.py